client/lib/analytics/tracker-buckets.ts

```diff
@@ -87,6 +87,18 @@ const checkGtmInit = (): boolean => {
 	return 'dataLayer' in window && 'google_tag_manager' in window;
 };
 
+/**
+ * Assign a user via Explat, or get the user's assignment. If a user is assigned to the treatment
+ * group, we do not load trackers or fire tracking events. Note, this does not involve Tracks, only
+ * third-party trackers.
+ */
+const mayWeTrackUserExplat = (): boolean => {
+	// Replace this with an explat assignment and return a boolean for whether we should load
+	// trackers and fire off tracking events in the signup.
+	// Check if we're in Stepper or Signup.
```

Comment:
Would checking if the path starts with `/setup` or `/start` help? I'm not sure where/when you end up calling this and in what context, so it may not work e.g. if during a redirection (Start to Stepper, etc.), may be ambiguous in that case

Reply:
I don't understand why the restriction tbh. Stepper is used for "signup" equally to Start (along with non-signup)